modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -73,16 +82,29 @@ def __init__(self, context, config, num_nodes, startup_timeout_sec, shutdown_tim
         self.main_java_class = main_java_class
         self.startup_timeout_sec = startup_timeout_sec
         self.shutdown_timeout_sec = shutdown_timeout_sec
+        self.thin_client_config = thin_client_config
+
+        if self.thin_client_config:
+            self.start_ignite = IgniteAwareService.StartIgnite.THIN_CLIENT
```

Comment:
start_ignite -> mode
Why do we need an instance variable here? 
It seems only a local variable required.